doc/_ext/ploomber_example.py

```diff
@@ -0,0 +1,92 @@
+from docutils import nodes
+from docutils.statemachine import ViewList
+from sphinx_tabs.tabs import TabsDirective, GroupTabDirective
+
+
+class PloomberExampleCliDirective(GroupTabDirective):
+
+    has_content = True
+
+    def run(self):
+
+        self.assert_has_content()
+
+        new_content = ViewList([
+            'CLI',
+            '',
+        ])
+
+        new_content.append(self.content)
+
+        self.content = new_content
+
+        node = super().run()
+
+        return node
+
+
+class PloomberExamplePythonDirective(GroupTabDirective):
+
+    has_content = True
+
+    def run(self):
+
+        self.assert_has_content()
+
+        # ploomber example config
+        # found with python -m sphinx.ext.intersphinx _build/html/objects.inv
+        ploomber_example_prepare_info_path = '../user-guide'\
+            '/using-python-api.html'
+
+        ref_node = nodes.reference(
+            '',
+            '',
+            internal=True,
+            # refname=app.config["ploomber_example_prepare_info_path"])
+            refuri=ploomber_example_prepare_info_path)
+
+        inner_node = nodes.emphasis("here", "here")
+
+        ref_node.append(inner_node)
+
+        note_node = nodes.note()
+
+        warning_text_node = nodes.paragraph(
+            text='Unlike the command line interface, the Ploomber Python '
```

Comment:
This might not matter but the two sentences are missing periods.

Reply:
fixed!